lib/temporal/workflow/future.rb

```diff
@@ -4,6 +4,7 @@ module Temporal
   class Workflow
     class Future
       attr_reader :target, :success_callbacks, :failure_callbacks
+      attr_accessor :child_workflow_execution_future # will complete when the child workflow has been completed
```

Comment:
This is specific to one case, but extends the interface for all Futures. Can we crease a subclass for child workflow future that gets initialized with this value? (So we don't need a writer exposed)

Reply:
Agreed, added this!